src/trinerdi/sc.sh

```diff
@@ -1,12 +1,13 @@
 alias t="basename \$(realpath .)" # Assumption: prints an alphanumeric string
-b(){
-	fl=-O2\ -fsanitize=address
-	[ a"$1" = a-d ] && fl=-g && v=valgrind
+e(){ # run on [problem].in*
+	if [ "$1" = -d ]; then fl=-g; v=valgrind
+	else fl=-O2\ -fsanitize=address; v=; fi
 	g++ -std=c++11 -lm -Wall -Wno-sign-compare -Wshadow $fl `t`.cpp -o `t`
 	for i in `t`.in*; do echo $i:; $v ./`t` < $i; done
 }
-n(){
-	mkdir "$1"
-	cd "$1"
-	cp -rn ../template.cpp "$1".cpp
+create(){
```

Comment:
I assume that we won't might not need to create a function for this during the contest if you can simply run the loop once. I understand that it makes sense to write it as a function here, though.

Reply:
Yes, I thought the same and figured this was way was the most convenient.